pkg/controller/daemon/update.go

```diff
@@ -367,12 +368,8 @@ func (dsc *DaemonSetsController) dedupCurHistories(ds *apps.DaemonSet, curHistor
 		}
```

Comment:
is there functional test coverage of this method to ensure this works properly?

Reply:
didn't find it... quickly scanned the 'daemonset_controller_test.go' and it looks like most of the tests are regarding the daemonset and pod instead of the controllerevision per my understanding.

Reply:
@liggitt wonder if anything else I can do to help move this forward?

Reply:
I was looking for a test exercising this change to make sure it works properly (either an existing test, or a newly added test)